src/renderer/components/NodeDocumentation/NodesList.tsx

```diff
@@ -79,6 +50,21 @@ export const NodesList = memo(() => {
         [byCategories, categories, scoreMap, searchQuery]
     );
 
+    const highestNode = useMemo(
+        () =>
+            byCategories
+                .get(categoriesByMaxNodeScore[0].name)
+                ?.sort(
+                    (a, b) => (scoreMap.get(b.schemaId) ?? 0) - (scoreMap.get(a.schemaId) ?? 0)
+                )[0] ?? undefined,
+        [byCategories, categoriesByMaxNodeScore, scoreMap]
+    );
+
+    useEffect(() => {
+        openNodeDocumentation(highestNode?.schemaId);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why ignore the warning?

Reply:
Because I only want it to run when that one changes

Reply:
It very much doesn't work properly with all the deps there, or at least it didn't back when I was doing this slightly differently. Maybe that changed though, so I'll try it again

Reply:
> It very much doesn't work properly with all the deps there

If you want to ensure that an effect only runs when a specific value changed, use a ref and store the last value in there. You can then do a `if (ref.current !== newValue)` inside the effect. Boilerplatey, but it's the only thing that is correct.